Process[surfacing_application-5] has died!

HELLO:
I tested it on Ubuntu 16.04, ROS-kinetic, PCL 1.8.0, VTK 7.1.1.
When I use GDB, it  shows:

```
[WARN] [1545377664.210452821]: Bezier::keepUpperPartofDilatedMesh: Computing normals...
[ INFO] [1545377666.796006527]: BezierGrindingSurfacing::generateTrajectory: Generating grinding trajectories
[ INFO] [1545377666.817293583]: BezierGrindingSurfacing::generateTrajectory: Generating extrication trajectories
[ERROR] [1545377666.836747402]: BezierGrindingSurfacing::generateTrajectory: Extrication stripper has 2 lines

Thread 1 "bezier_applicat" received signal SIGSEGV, Segmentation fault.
0x000000000042a5c2 in _mm256_load_pd (__P=0x7fffffffbb10)
    at /usr/lib/gcc/x86_64-linux-gnu/5/include/avxintrin.h:836
836	  return *(__m256d *)__P;
(gdb) 
```

--- a/bezier_library/src/bezier_surfacing.cpp
+++ b/bezier_library/src/bezier_surfacing.cpp
@@ -154,7 +154,7 @@ std::string BezierGrindingSurfacing::generateTrajectory(EigenSTL::vector_Affine3
         ROS_ERROR_STREAM(
             "BezierGrindingSurfacing::generateTrajectory: Grinding stripper has " << stripper->GetOutput()->GetNumberOfLines() << " lines");
         // FIXME Handle this case!
-        //return "Grinding stripper has more than 1 line (mesh has a hole). Not implemented yet!";
+        return "Grinding stripper has more than 1 line (mesh has a hole). Not implemented yet!";
       }
     }
 

